src/components/Icons/ActionDot.icon.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import * as React from 'react'
+import Svg, { G, Path, Defs, ClipPath } from 'react-native-svg'
+import { COLORS } from '../../styles/themes'
+import { DEFAULT_ICON_SIZE } from './constants'
+import { getScale, getViewBox } from './helper'
+
+export const ActionDot = ({ size = DEFAULT_ICON_SIZE, color = COLORS.black }) => (
+  <Svg
+    xmlns='http://www.w3.org/2000/svg'
+    width={DEFAULT_ICON_SIZE}
+    height={DEFAULT_ICON_SIZE}
+    viewBox={getViewBox(size)}
+    transform={getScale(size, DEFAULT_ICON_SIZE)}
+    fill='none'
+  >
+    <G clipPath='url(#a)' opacity={0.7}>
```

Comment:
Does it make sense to have this opacity hardcoded? Or maybe receive this in the props?